src/mgr.sol

```diff
@@ -128,43 +131,43 @@ contract TinlakeManager {
 
     // Tinlake components
     GemLike      public gem;
-    GemLike      public tin;
+    GemLike      public rwaToken;
     RedeemLike   public pool;
 
+    // MIP21 RWAUrn
+    MIP21UrnLike public urn;
```

Comment:
where is this given a value?

Reply:
In the `file` method.
 Because we have bi-directional dependency with RwaUrn if the mgr is the operator and output-conduit

Reply:
only in file I suppose